lib/storage/bucket.js

```diff
@@ -235,6 +237,11 @@ Bucket.prototype.file = function(name) {
  *     bucket.getFiles(nextQuery, function(err, files, nextQuery) {});
  *   }
  *
+ *   // The prefixes property contains any prefixes returned in the response.
```

Comment:
Silly: can you put quotes around `prefixes`? I was reading too quickly and didn't realize it was a property until I realized that there was a verb-noun disagreement with "prefixes" and "contains". It seems you have quotes around the properties below, so it'd be helpful to have them here too. :sparkles: